job.go

```diff
@@ -14,21 +14,21 @@ import (
 type Job struct {
 	sync.RWMutex
 	jobFunction
-	interval          int            // pause interval * unit between runs
-	duration          time.Duration  // time duration between runs
-	unit              schedulingUnit // time units, ,e.g. 'minutes', 'hours'...
-	startsImmediately bool           // if the Job should run upon scheduler start
-	atTime            time.Duration  // optional time at which this Job runs when interval is day
-	startAtTime       time.Time      // optional time at which the Job starts
-	error             error          // error related to Job
-	lastRun           time.Time      // datetime of last run
-	nextRun           time.Time      // datetime of next run
-	scheduledWeekday  *time.Weekday  // Specific day of the week to start on
-	dayOfTheMonth     int            // Specific day of the month to run the job
-	tags              []string       // allow the user to tag Jobs with certain labels
-	runCount          int            // number of times the job ran
-	timer             *time.Timer    // handles running tasks at specific time
-	cronSchedule      cron.Schedule  // stores the schedule when a task uses cron
+	interval          int             // pause interval * unit between runs
+	duration          time.Duration   // time duration between runs
+	unit              schedulingUnit  // time units, ,e.g. 'minutes', 'hours'...
+	startsImmediately bool            // if the Job should run upon scheduler start
+	atTime            time.Duration   // optional time at which this Job runs when interval is day
+	startAtTime       time.Time       // optional time at which the Job starts
+	error             error           // error related to Job
+	lastRun           time.Time       // datetime of last run
+	nextRun           time.Time       // datetime of next run
+	scheduledWeekday  []*time.Weekday // Specific days of the week to start on
```

Comment:
Now that we have a slice here I don't think we still need to be working with pointers anymore, what do you think? That'll make implementation simpler and safer.

Reply:
Instead of check if `scheduledWeekday` equals `nil`, now I just check the length. Done!